CMakeLists.txt

```diff
@@ -101,13 +102,9 @@ endif()
 # Export CMake for installation downstream
 
 # Build list of CMake files to install
-set(_cmake_files)
-if(VecGeom_VERSION VERSION_LESS 1.2.10)
-  # Only install RDC utils if not available through vecgeom
-  list(APPEND _cmake_files
-    "${PROJECT_SOURCE_DIR}/cmake/CudaRdcUtils.cmake"
-  )
-endif()
+set(_cmake_files
+  "${PROJECT_SOURCE_DIR}/cmake/external/CudaRdcUtils.cmake"
```

Comment:
We could consider:
```
set(_cmake_files ${_LOCAL_RDCUTILS_FILENAME})
```
for consistency (or reverse)
```
include("${PROJECT_SOURCE_DIR}/cmake/external/CudaRdcUtils.cmake")
```

Reply:
Oh yeah. I think it's probably safer and clearer to repeat the full path than have a "nonlocal" variable: changing the varaiable name would result in a silent failure (no file installed) whereas changing the path would tell us the problem.